crates/bevy_gizmos/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 //!
 //! See the documentation on [Gizmos](crate::gizmos::Gizmos) for more examples.
 
-/// Label for the the render systems handling the
+/// Label for the render systems handling the
```

Comment:
It's potentially unrelated, but I think this comment is incomplete and doesn't say all that it should. Or there is just a random redundant `the` at the end of it. Or both.

Reply:
Thanks for the review. Should I revert the changes here?

Reply:
```suggestion
/// System set label for the systems handling the rendering of gizmos.
```